lib/container/container.go

```diff
@@ -133,6 +133,25 @@ func WaitContainer(id string) (int, error) {
 	return client.WaitContainer(id)
 }
 
+func getNewDockerClient() (*docker.Client, error) {
+	if os.Getenv("DOCKER_TLS_VERIFY") == "1" {
+		path := os.Getenv("DOCKER_CERT_PATH")
```

Comment:
should do some checks, i do something similar to all this at https://github.com/peter-edge/go-dockerutils/blob/master/dockerutils.go#L84, it checks to make sure the files exist, etc